week-3/B-array-some/exercise.js

```diff
@@ -7,11 +7,21 @@
 */
 
 var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
+var pairsByIndex = [[0, 3], [1, 2], [2, 1], [3, 0]];
```

Comment:
You weren't supposed to change this input :-)  Make sure the nullValue function below actually checks for nulls!